pkg/windows/windows.go

```diff
@@ -256,23 +263,28 @@ type windows struct {
 	log      logr.Logger
 	// defaultShellPowerShell indicates if the default SSH shell is PowerShell
 	defaultShellPowerShell bool
+	// filesToTransfer is the map of files needed for the windows VM
+	filesToTransfer map[*payload.FileInfo]string
 }
 
 // New returns a new Windows instance constructed from the given WindowsVM
-func New(clusterDNS string, instanceInfo *instance.Info, signer ssh.Signer) (Windows, error) {
+func New(clusterDNS string, instanceInfo *instance.Info, signer ssh.Signer, platform *config.PlatformType) (Windows, error) {
 	log := ctrl.Log.WithName(fmt.Sprintf("wc %s", instanceInfo.Address))
 	log.V(1).Info("initializing SSH connection")
 	conn, err := newSshConnectivity(instanceInfo.Username, instanceInfo.Address, signer, log)
 	if err != nil {
 		return nil, fmt.Errorf("unable to setup VM %s sshConnectivity: %w", instanceInfo.Address, err)
 	}
 
+	files, err := createPayload(platform)
```

Comment:
need to check the error value
```suggestion
	files, err := createPayload(platform)
	if err != nil {
		return nil, fmt.Errorf("unable to create payload: %w", err)
	}
```

Reply:
> return fmt.Errorf("error getting list of files to transfer: %w", err)

I would rather stick to the [existing](https://github.com/openshift/windows-machine-config-operator/pull/1778/commits/956a384bcd1b1421ef04531545af7258e9b3b130#diff-4d8fe3bb66a872f0e4a277f820d60d2efa722f09aa89516be74b98c06b57f3bdL657) message for consistency.